spec/tail_spec.rb

```diff
@@ -174,5 +176,47 @@
         expect { |b| subject.subscribe(&b) }.not_to yield_control
       end
     end
+
+    let(:new_file_path) { File.join(directory, "2.log") }
+
+    context "when a new file is later added to the directory" do
+      before do
+        File.open(new_file_path, "wb") { |file|  file.write("line2.1\nline2.2\n") }
+        should_yield.push [new_file_path, "line2.1"]
+        should_yield.push [new_file_path, "line2.2"]
+      end
+
+      it "reads new lines off the new file" do
+        expect { |b| subject.subscribe(&b) }.to yield_successive_args(*should_yield)
+      end
+
+      context "and when the sincedb path is not given" do
```

Comment:
Well position could be beginning or end and the context will not know that.  I will hardcode it to `beginning` and change the text.